scripts/filter-media.md

```diff
@@ -0,0 +1,44 @@
+# Filter-Media Script
```

Comment:
add a link back to the list of the scripts (something similar is done in each individual endpoint file toward the endpoint.md list)